### PR TITLE
Packaging

### PR DESCRIPTION
**Pull request by LucsT**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,122 +1,124 @@
...
 
 module.exports = new BaseKonnector(start)
 
-// The start function is run by the BaseKonnector instance only when it got all the account
-// information (fields). When you run this connector yourself in "standalone" mode or "dev" mode,
-// the account information come from ./konnector-dev-config.json file
 async function start(fields) {
   log('info', 'Authenticating ...')
-  await authenticate(fields.login, fields.password)
+  await authenticate(fields.login, fields.password, fields.dateOfBirth)
```
Comment (doubleface) [inline src/index.js]: The manifest says it's "dob" instead of "dateOfBirth"

File: src/index.js [hunk 2]
```diff
@@ -1,122 +1,124 @@
...
-        parse: title => `${title}.jpg`
-      }
-    },
-    'article'
+async function parseDeposits() {
+  const items = await getDeposits()
+  let entries = []
+  for (let i in items) {
+    const date =  moment(items[i].date)
+    const html = '<body><table>'+
```
Comment (aenario) [inline src/index.js]: I was about to comment you could also have used directly pdfjs API to build the pdf, but when i started to write it, I saw it was going to be much longer, so good idea !

File: src/index.js [hunk 3]
```diff
@@ -1,122 +1,124 @@
...
+        statusFilter: 'deposit',
+        isFreeAccount: false
       }
+    })
+    // Eliminate empty last page if needed
+    if (res.transactionsHistoryItems) {
+      list = list.concat(res.transactionsHistoryItems)
     }
-  })
+    if (!res.phHasNext || res.phHasNext === false) {
```
Comment (aenario) [inline src/index.js]: if `phHasNext === false`, the first part will always be false.

Comment (LucsT) [reply src/index.js]: Yes, but in fact, it is no more necessary due to the previous test line to trash an empty.
For the last page, the phHasNext is not existing.
